clang_3.8/Dockerfile

```diff
@@ -9,7 +9,8 @@ ENV LLVM_VERSION=3.8 \
     CMAKE_CXX_COMPILER=clang++ \
     PYENV_ROOT=/opt/pyenv \
     PATH=/opt/pyenv/shims:${PATH} \
-    CONAN_CLANG_VERSIONS=3.8
+    CONAN_CLANG_VERSIONS=3.8 \
+    CONAN_ARCHS=x86_64
```

Comment:
Why? We can cross-compile to x86 using this image. Same for other images on this commit.

Reply:
I can put `CONAN_ARCHS=x86_64,x86` if your prefer. Environment variables can be set from CI anyway, It's just a matter of choosing the most common use case. It looked more consistent to have only x86_64, with conan-center-index supporting x86_64 only, but I have no strong feeling about this.

Reply:
Yeah, but Conan Center Index is not the only one who uses these docker images. We are using for Bincrafters too. Of course, we dropped x86 support, I would say someone will complain about this. Please, don't touch CONAN_ARCHS